### ISSUE TITLE
nextage_moveit_config depends on hironx_moveit_config

### ISSUE THREAD

**Issue by k-okada** [tork-a/rtmros_nextage#71]
At this moment, nextage_moveit_config is not depend hironx_moveit_config. but @youtalk reporeted we need that one. @130s Can you confirm this? we should have to either add depend to hironx_moveit_config or remove/replace something.

```
/opt/ros/hydro/share/nextage_moveit_config$ grep -r hironx *
launch/NextageOpen_moveit_controller_manager.launch:       cp `rospack find hironx_moveit_config`/launch/HiroNX_moveit_controller_manager.launch 
```

**Comment by 130s** [comment]
I'll have to spend more time carefully later but with a quick look this is strange; `NextageOpen_moveit_controller_manager.launch` seems to be the only place where the term `hironx` appears in `nextage_moveit_config` package, but that launch file isn't used anywhere.

```
/rtmros_nextage/nextage_moveit_config$ cd ..
/rtmros_nextage$ grep -r NextageOpen_moveit_controller_manager.launch .
/rtmros_nextage$ 
```

**Comment by 130s** [comment]
tl;dr @youtalk What were the **related packages**? 
 You [mentioned here](https://github.com/tork-a/rtmros_nextage/issues/67#issuecomment-44917552):

```
After installing ros-hydro-hironx-moveit-config related packages,
```

I guess those "related" packages got installed because of the dependency that only `hironx_moveit_config` depends on (ie. `nextage_moveit_config` doesn't) and that might be reason why installing `hironx_moveit_config` solved the issue.

---

I can't replicate the reported issue in https://github.com/tork-a/rtmros_nextage/issues/67, but I got a bit more clue about the possible cause.
1. [nextage_moveit_config/move_group.launch](https://github.com/tork-a/rtmros_nextage/blob/f8f036565f5248c68a3459f47e53b6e0abae730d/nextage_moveit_config/launch/move_group.launch#L22) calls `nextage_moveit_config/trajectory_execution.launch`
2. In [nextage_moveit_config/trajectory_execution.launch](https://github.com/tork-a/rtmros_nextage/blob/f8f036565f5248c68a3459f47e53b6e0abae730d/nextage_moveit_config/launch/trajectory_execution.launch#L11), `nextage_moveit_config/NextageOpen_moveit_controller_manager.launch` is called (`NextageOpen` is dynamically filled in by `arg` attribute).
3. [nextage_moveit_config/NextageOpen_moveit_controller_manager.launch](%28https://github.com/tork-a/rtmros_nextage/blob/f8f036565f5248c68a3459f47e53b6e0abae730d/nextage_moveit_config/launch/NextageOpen_moveit_controller_manager.launch%29) file seems to have been copied from `hironx_moveit_config`/launch/HiroNX_moveit_controller_manager.launch`. But the content of this file in 2 packages appear to be the same.
   
   Here we lost a clue, but this also means that also indicates that `NextageOpen_moveit_controller_manager.launch` might irrelevant to the reported issue.
4. Looking at `package.xml` of both `nextage_moveit_config` and `hironx_moveit_config`, `run_depend` is quite different:
   
   From [nextage_moveit_config](https://github.com/tork-a/rtmros_nextage/blob/f8f036565f5248c68a3459f47e53b6e0abae730d/nextage_moveit_config/package.xml#L19)

```
  <run_depend>moveit_ros_move_group</run_depend>
  <build_depend>nextage_description</build_depend>
  <run_depend>nextage_description</run_depend>
```

  From [hironx_moveit_config](https://github.com/start-jsk/rtmros_hironx/blob/c861995a7df8f6240afec4e6b2bac21ced49bc3a/hironx_moveit_config/package.xml)

```
  <build_depend>collada_urdf</build_depend>
  <build_depend>hironx_ros_bridge</build_depend>
  <build_depend>moveit_ros_move_group</build_depend>

  <run_depend>hironx_ros_bridge</run_depend>
  <run_depend>moveit_planners</run_depend>
  <run_depend>moveit_ros</run_depend>
  <run_depend>pr2_moveit_plugins</run_depend>
```

Again, what were the '''related packages'''?

**Comment by youtalk** [comment]
The "related packages" mentioned in https://github.com/tork-a/rtmros_nextage/issues/67#issuecomment-44917552 are `ros-hydro-moveit-*` packages. All of them are as follow. Of course, I had already installed `ros-hydro-nextage-moveit-config` before.

> ``` sh
> $ sudo apt-get install ros-hydro-hironx-moveit-config
> Reading package lists... Done
> Building dependency tree       
> Reading state information... Done
> The following extra packages will be installed:
>   libmongo-client-dev libmongo-client0 mongodb-clients mongodb-dev mongodb-server
>   ros-hydro-household-objects-database-msgs ros-hydro-manipulation-msgs
>   ros-hydro-moveit-planners ros-hydro-moveit-planners-ompl ros-hydro-moveit-ros
>   ros-hydro-moveit-ros-benchmarks ros-hydro-moveit-ros-benchmarks-gui
>   ros-hydro-moveit-ros-manipulation ros-hydro-moveit-ros-planning-interface
>   ros-hydro-moveit-ros-robot-interaction ros-hydro-moveit-ros-visualization
>   ros-hydro-moveit-ros-warehouse ros-hydro-ompl ros-hydro-pr2-moveit-plugins
>   ros-hydro-warehouse-ros
> ```

I've checked the repository codes and yes, I also think this problem causes by the difference of both `package.xml` files.

**Comment by k-okada** [comment]
I see, I think we need pr2_moveit_plugins, at least.
create PR for that
https://github.com/tork-a/rtmros_nextage/pull/73

**Comment by 130s** [comment]
Fixed via https://github.com/tork-a/rtmros_nextage/pull/73
